src/shared/components/card/CardImage.tsx

```
@@ -0,0 +1,54 @@
+  return (
+    <div
+      className={styles.cardcontainer({ state: visualState })}
+      onMouseDown={handleMouseDown}
+      onMouseUp={handleMouseUp}
+    >
+      <img
+        src={example}
+        className={disabled ? styles.disabledcardimg : styles.cardimg}
+      />
+      {visualState !== 'disabled' && (
+        <button className={styles.checkbox({ state: visualState })}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)